Library/ViewModels/ProjectSummaryCarouselCellViewModel.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+import KsApi
+import Prelude
+import ReactiveSwift
+
+public protocol ProjectSummaryCarouselCellViewModelInputs {
+  func configure(with item: ProjectSummaryEnvelope.ProjectSummaryItem)
+}
+
+public protocol ProjectSummaryCarouselCellViewModelOutputs {
+  var body: Signal<String, Never> { get }
+  var title: Signal<String, Never> { get }
+}
+
+public protocol ProjectSummaryCarouselCellViewModelType {
+  var inputs: ProjectSummaryCarouselCellViewModelInputs { get }
+  var outputs: ProjectSummaryCarouselCellViewModelOutputs { get }
+}
+
+public final class ProjectSummaryCarouselCellViewModel: ProjectSummaryCarouselCellViewModelType,
+  ProjectSummaryCarouselCellViewModelInputs, ProjectSummaryCarouselCellViewModelOutputs {
+  public init() {
+    let item = self.itemProperty.signal
+      .skipNil()
+
+    self.body = item
+      .map(\.response)
+
+    self.title = item
+      .map(\.question)
+      .map(titleText(for:))
+  }
+
+  private let itemProperty = MutableProperty<ProjectSummaryEnvelope.ProjectSummaryItem?>(nil)
+  public func configure(with item: ProjectSummaryEnvelope.ProjectSummaryItem) {
+    self.itemProperty.value = item
+  }
+
+  public let body: Signal<String, Never>
+  public let title: Signal<String, Never>
+
+  public var inputs: ProjectSummaryCarouselCellViewModelInputs { return self }
+  public var outputs: ProjectSummaryCarouselCellViewModelOutputs { return self }
+}
+
+private func titleText(
+  for question: ProjectSummaryEnvelope.ProjectSummaryItem.ProjectSummaryQuestion
+) -> String {
+  switch question {
+  case .whatIsTheProject:
+    return localizedString(
```

Comment:
Apparently this will be an english-only experiment so will confirm whether these need to be added for translation.